examples/rules.mk

```diff
@@ -30,10 +30,15 @@ INC += $(TOP)/src
 
 #
 CFLAGS += $(addprefix -I,$(INC))
-LDFLAGS += $(CFLAGS) -fshort-enums -Wl,-T,$(TOP)/$(LD_FILE) -Wl,-Map=$@.map -Wl,-cref -Wl,-gc-sections -specs=nosys.specs -specs=nano.specs
+ifeq ($(BOARD), msp_exp430f5529lp)
+  LDFLAGS += $(CFLAGS) -fshort-enums -Wl,-T,$(TOP)/$(LD_FILE) -Wl,-Map=$@.map -Wl,-cref -Wl,-gc-sections
+else
+  LDFLAGS += $(CFLAGS) -fshort-enums -Wl,-T,$(TOP)/$(LD_FILE) -Wl,-Map=$@.map -Wl,-cref -Wl,-gc-sections -specs=nosys.specs -specs=nano.specs
+endif
+LDFLAGS += $(addprefix -L,$(LDINC))
```

Comment:
I think we should move `LDFLAGS += $(addprefix -L,$(LDINC))` to board.mk like https://github.com/hathach/tinyusb/blob/master/hw/bsp/pca10056/board.mk#L24 
since it is not as nearly universal like C header include path